### ISSUE TITLE
Address location buttons not reading content on screen readers 

### ISSUE THREAD

**Issue by bpdesigns** [18F/e-QIP-prototype#882]
When Mac's Voiceover screenreader reads the buttons related to the question “This address is: | In the United States | APO/FPO/DPO | Outside of the United States“ | are not labeled for screenreader.” the screenreader just calls out “[object Object]“ instead of the text that is visible in those button options. 

I expect that issue #822 will resolve this.
